.circleci/config.yml

```diff
@@ -22,9 +22,9 @@ workflows:
       - contract-size:
           requires:
             - compile
-      - deploy-test:
-          requires:
-            - compile
+      # - deploy-test:
```

Comment:
I commented this, not sure why we deploy using CI?!

Reply:
better to just remove the lines than leave them commented there